app/Http/Controllers/Auth/AuthController.php

```diff
@@ -33,9 +40,17 @@ public function login()
 
     public function loginPost()
     {
-        \Auth::attempt(request()->only(['email', 'password']), request('remember'));
+        if (Auth::attempt(['email' => request('email'), 'password' => request('password'), 'status' => 'active'], request('remember'))) {
+            Notifications::success('Successfully login', 'top');
```

Comment:
Also why not to leave it without group? It will assign group `0`. And you can use it by default in top.